RoddyCore/src/de/dkfz/roddy/knowledge/methods/GenericMethod.groovy

```diff
@@ -554,8 +554,9 @@ class GenericMethod {
         if (!fileParameter.checkFile.evaluate(context))
             bf.setAsTemporaryFile()
 
-        if (allInputFiles.size() > 1)
-            bf.setParentFiles(allInputFiles, true)
+        // Should already be done with the helper.
```

Comment:
Why keep the comment?

Reply:
As a reminder for you, the reviewer to look over it.

Reply:
Done in the constructor using the "ConstructionHelper" parameter object:

de.dkfz.roddy.knowledge.files.BaseFile#BaseFile(de.dkfz.roddy.knowledge.files.BaseFile.ConstructionHelperForBaseFiles)

Note that there `this.path` is set to the saved old value, if the filename-pattern resolution fails. In this constructor/new instance context a resetting to on old value is equivalent to setting to the default value and is not necessary.